src/Hudl.FFmpeg/Hudl.FFmpeg.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
Are these libs getting pushed to Nuget? if so you may need to add some Assembly attributes here like `AssemblyFileVersion`, also a version for the nuget package?